daal4py/sklearn/_device_offload.py

```diff
@@ -14,26 +14,159 @@
 # limitations under the License.
 # ==============================================================================
 
+import logging
+import sys
+from collections.abc import Iterable
 from functools import wraps
 
+import numpy as np
+
+from ._config import _get_config
+
 try:
```

Comment:
Actually it doesn't do replication. I have moved necessary primitives to daal4py level to avoid code duplication and import of sklearnex on daal4py side